akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -55,6 +60,16 @@
      */
     ResponseEntity entity();
 
+    /**
+     *  Drains entity stream of this message
+     */
+    DiscardedEntity discardEntityBytes(Materializer materializer);
+
+    interface DiscardedEntity {
```

Comment:
please add documentation;

final class

"Represents the the currently being-drained HTTP Entity which triggers completion of the contained Future once the entity has been drained for the given HttpMessage completely. "